### ISSUE TITLE
Mention JupyterLab Desktop on "Install" and "Try" pages

### ISSUE THREAD

**Issue by JasonWeill** [jupyter/jupyter.github.io#737]
The JupyterLab Desktop app provides a streamlined installer so that users less familiar with CLI tools can use notebooks on a personal computer. The "Try" and "Install" pages mention some of our apps, but not JupyterLab Desktop.

I spoke with @mbektas and he was OK with us promoting Desktop in more places, including on our web site. Let's add it as a first-class project on jupyter.org.

**Comment by krassowski** [comment]
I mean, yes but it would be so nice if we could get the issue with it being flagged as a security risk on widows resolved first. For the sake of project reputation and promoting good practices I think we should not be telling people "your OS will flag it as a potential virus but don't worry you can ignore it".

https://github.com/jupyterlab/jupyterlab-desktop/issues/202#issuecomment-1595703435

**Comment by JasonWeill** [comment]
This is a duplicate of #589, but let's leave it open, in the hope that we can get this one done.

Related issue on JupyterLab Desktop repo: https://github.com/jupyterlab/jupyterlab-desktop/issues/275

**Comment by JasonWeill** [comment]
Still blocked on code signing certificate, as @krassowski mentioned earlier: https://github.com/jupyterlab/jupyterlab-desktop/issues/202

**Comment by hxrshxz** [comment]
kindly have a look at the PR and tell if any other changes need to me made
